base/uk.ac.stfc.isis.ibex.ui.journalviewer/src/uk/ac/stfc/isis/ibex/ui/journalviewer/JournalViewerView.java

```diff
@@ -153,6 +155,11 @@ public void createPartControl(final Composite parent) {
         progressBar = new ProgressBar(searchControls, SWT.INDETERMINATE);
         progressBar.setMaximum(80);
         progressBar.setLayoutData(new RowData(100, SWT.DEFAULT));
+        
+        error = new Label(searchControls, SWT.NONE);
+        error.setForeground(parent.getDisplay().getSystemColor(SWT.COLOR_RED));
+        error.setLayoutData(new RowData(200, SWT.DEFAULT));
+        error.setText("placeholder");
```

Comment:
What is "placeholder" string used for? Could it just be empty?

Reply:
Removed, seems like we do not need it.